pkg/config/config.go

```diff
@@ -680,6 +680,7 @@ func InitConfig(config Config) {
 	// enabling/disabling the environment variables & command scrubbing from the container specs
 	// this option will potentially impact the CPU usage of the agent
 	config.BindEnvAndSetDefault("orchestrator_explorer.container_scrubbing.enabled", true)
+	config.BindEnvAndSetDefault("orchestrator_explorer.custom_sensitive_words", []string{})
```

Comment:
this one was missing to let viper auto register it under `DD_ORCHESTRATOR_EXPLORER_CUSTOM_SENSITIVE_WORDS`